### PR TITLE
ui - issue view - fix icon position

### PR DESCRIPTION
**Pull request by Cherrg**
issue view - fix icon position
- move style from template to css
- add bullets to: key, lock, circle-slash, comment

Signed-off-by: Michael Gnehr <michael@gnehr.de>

### Images

| Old | New | Old dark | New dark | dark (with #7355) |
|--|--|--|--|--|
| grafik | grafik  | grafik | grafik | grafik  |

## other commits:
* (2nd) add border to symbols without bullet

| result (2nd) | change color to display added border (2nd) | update top alignment | |
|--|--|--|--|
| grafik | grafik| grafik  | grafik |

### CODE REVIEW

File: public/less/_repository.less [hunk 1]
```diff
@@ -899,6 +899,10 @@
                 margin: 15px 0 15px 79px;
                 padding-left: 25px;
 
+                & > .octicon:not(.issue-symbol) {
+                    text-shadow: -2px 0 #ffffff, 0 2px #ffffff, 2px 0 #ffffff, 0 -2px #ffffff;
```
Comment (silverwind) [inline public/less/_repository.less]: Could use shorthand colors like `#fff`.

**Comment by silverwind** [comment]
There's some mixed tab/space indentation in those template files, some probably existed before this commit. Maybe you wanna fix them:

````
.git/rebase-apply/patch:133: space before tab in indent.
     	</div>
.git/rebase-apply/patch:137: space before tab in indent.
	    	<span class="octicon octicon-{{.Review.Type.Icon}} issue-symbol"></span>
.git/rebase-apply/patch:138: space before tab in indent.
	    	<a class="ui avatar image" href="{{.Poster.HomeLink}}">
.git/rebase-apply/patch:139: space before tab in indent.
	    		<img src="{{.Poster.RelAvatarLink}}">
.git/rebase-apply/patch:140: space before tab in indent.
	    	</a>
warning: 1 line applied after fixing whitespace errors.
````

**Comment by silverwind** [comment]
Suggestion: Align icon vertically with avatar, if it's not too hard:
 
Screenshot 2019-07-04 at 17 47 06

**Review by silverwind — APPROVED** [review]
LGTM with above suggestions.

File: public/less/_repository.less (hunk 1 above)
Comment (Cherrg) [reply public/less/_repository.less]: strange:
- i know this is possible in css
- but css linter (make css) always said i should change this to full length
- if i change this to short form in .less, currently there is no warning...

-> in rendered css there is already the short form

Comment (silverwind) [reply public/less/_repository.less]: > but css linter (make css) always said i should change this to full length

Check if your linter actually uses [stylelint](https://github.com/stylelint/stylelint) or something else. We use stylelint and explicitely disable the [color-hex-length](https://github.com/go-gitea/gitea/blob/49ee9d27718c31d5ab4fe9543fefc90cd40d4405/.stylelintrc#L5) rule so the linter has no preference currently (but I want to eventually enable it to the short form).

> in rendered css there is already the short form

That's the minifier ([clean-css](https://github.com/jakubpawlowicz/clean-css)) optimizing for size.

**Comment by Cherrg** [comment]
> There's some mixed tab/space indentation in those template files, some probably existed before this commit. Maybe you wanna fix them:
> 
> ```
> .git/rebase-apply/patch:133: space before tab in indent.
>      	</div>
> .git/rebase-apply/patch:137: space before tab in indent.
> 	    	<span class="octicon octicon-{{.Review.Type.Icon}} issue-symbol"></span>
> .git/rebase-apply/patch:138: space before tab in indent.
> 	    	<a class="ui avatar image" href="{{.Poster.HomeLink}}">
> .git/rebase-apply/patch:139: space before tab in indent.
> 	    		
> .git/rebase-apply/patch:140: space before tab in indent.
> 	    	</a>
> warning: 1 line applied after fixing whitespace errors.
> ```

which command show me this list?

File: public/less/_repository.less (hunk 1 above)
Comment (Cherrg) [reply public/less/_repository.less]: i've changed to short form

**Comment by Cherrg** [comment]
> Suggestion: Align icon vertically with avatar, if it's not too hard:
> 
> Screenshot 2019-07-04 at 17 47 06

should be ok now

**Comment by silverwind** [comment]
> which command show me this list?

You can see them during `git apply`:

````
$ curl -sL https://github.com/go-gitea/gitea/pull/7354.patch | git apply
<stdin>:142: space before tab in indent.
     	</div>
<stdin>:146: space before tab in indent.
	    	<span class="octicon octicon-{{.Review.Type.Icon}} issue-symbol"></span>
<stdin>:147: space before tab in indent.
	    	<a class="ui avatar image" href="{{.Poster.HomeLink}}">
<stdin>:148: space before tab in indent.
	    		<img src="{{.Poster.RelAvatarLink}}">
<stdin>:149: space before tab in indent.
	    	</a>
warning: squelched 20 whitespace errors
warning: 18 lines applied after fixing whitespace errors.
````

**Comment by silverwind** [comment]
Icon alignment looking good, thanks.

**Review by kolaente — APPROVED** [review]

**Pull request merged**